README.md

```diff
@@ -310,6 +310,7 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 * [Dropwizard](https://dropwizard.github.io/dropwizard/) - Opinionated framework for setting up modern web applications, includes Jetty, Jackson, Jersey and Metrics.
 * [Feign](https://github.com/Netflix/feign) - Java to HTTP client binder inspired by Retrofit, JAXRS-2.0, and WebSocket.
 * [Jersey](https://jersey.java.net/) - JAX-RS reference implementation.
+* [Qbit](https://github.com/advantageous/qbit) - The Microservice Lib for Java - JSON, REST, WebSocket, Speed!
```

Comment:
The description is copy/paste and doesn't really describe much. Can you change it to "Reactive programming library for building microservices"?